Homeowrk4.ipynb

FrankieChik opened the pull request.

jdamerow committed:
Added homework for week 7

Commit:
Homeowrk4.ipynb

jdamerow commented on Homeowrk4.ipynb:
```diff
@@ -0,0 +1,281 @@
+    "        self.filesize=filesize\n",
+    "    def load_text(self):\n",
+    "        print(\"loading\" + self.path_to_file) \n",
+    "    def smaller_size(filesize, current_size):\n",
```
class methods have to have `self` as first argument; `self` represents the object itself, so you would use self to get the filesize of the current object and the second argument would be the other text object ("a method that accepts another text object")

jdamerow commented on Homeowrk4.ipynb:
```diff
@@ -0,0 +1,281 @@
+    "t3=Text(\"/path/to/my/file\", \"doc\", 45)\n",
+    "t3.load_text()\n",
+    "print(t3.filetype)\n",
+    "print(t3.filesize)    "
```
The assignment asks to put the text objects "in a list. For each element in the list print filepath, type of file, and filesize."

jdamerow commented on Homeowrk4.ipynb:
```diff
@@ -0,0 +1,281 @@
+    "    def __init__(self, collectionspath):\n",
+    "        self.path_to_collections=collectionspath\n",
+    "    def load_collections(self):\n",
+    "        print(\"load collections\" + self.path_to_collections)\n",
```
class is missing: "It should have a list of collections objects."

jdamerow commented on Homeowrk4.ipynb:
```diff
@@ -0,0 +1,281 @@
+    "        self.path_to_file=filepath\n",
+    "        self.yeara=yeara\n",
+    "    def load_filesa(self):\n",
+    "        print(\"load filesa\" + self.path_to_file)\n",
```
missing: "a list of Text objects."

jdamerow commented on Homeowrk4.ipynb:
```diff
@@ -0,0 +1,281 @@
+    "        self.yearc=yearc\n",
+    "    def load_filesc(self):\n",
+    "        print(\"load filesc\" + self.path_to_file)\n",
+    "        \n",
```
why are there three collection classes that all look the same?

FrankieChik committed:
Merge pull request #14 from diging-training/master

Added homework for week 7

jdamerow committed:
added homework for week 8

FrankieChik committed:
Create README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

Commit:
Homework-05.ipynb

jdamerow commented on Homework-05.ipynb:
```diff
@@ -0,0 +1,197 @@
+    "    f.write(json_object) \n",
+    "with open('sample.json', 'r') as f:\n",
+    "    output = json.load(f)\n",
+    "print(output)"
```
The assignment was "Then let the user enter new entries (consisting of at least a title, an author, and an item type)", but other than that good.

jdamerow commented on README.md:
```diff
@@ -0,0 +1,17 @@
+  * *Zigao*2
+
+3. Conclusion  
+<https://www.history.com/this-day-in-history/last-emperor-of-china-abdicates>
```
usually, you use `[link text](link)` to create links.

jdamerow replied:
and the assignment asked for multiple paragraphs of text. other than that good.

FrankieChik committed:
Merge pull request #16 from diging-training/master

added homework for week 8

FrankieChik committed:
Create index.md

FrankieChik committed:
Create secondfile

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update secondfile

jdamerow commented on doc-folder/index.md:
```diff
@@ -0,0 +1,65 @@
+if loser==1:  
+    print("plater 1 is the loser.")  
+elif loser==2:  
+    print("player 2 is the loser.") 
```
there is a specific markup to display code properly in Markdown files.

jdamerow reviewed (changes requested):
Hey Frankie, is that correct that you didn't work on assignment 2 and 3?

FrankieChik commented:
Hi Julia,
I should have finished the assignments 2 and 3.
Best,
Frankie

> Julia Damerow <notifications@github.com> 於 2020年3月4日 上午8:15 寫道：
> 
> @jdamerow requested changes on this pull request.
> 
> In Homework-05.ipynb <https://github.com/FrankieChik/python-for-the-humanities-homework/pull/13#discussion_r387729037>:
> 
> > +    "    {'title': 'Confucius beyond the Analects', \n",
> +    "    'authors': 'Michael Hunter', \n",
> +    "    'itemType': 'Monograph'\n",
> +    "    },\n",
> +    "    {'title': 'Dividing the realm in order to govern: the spatial organization of the Song state, 960-1276 CE',\n",
> +    "     'authors': 'Ruth Mostern',\n",
> +    "     'itemType': 'Monograph'\n",
> +    "    }\n",
> +    " ]\n",
> +    "            }\n",
> +    "json_object = json.dumps(dictionary, indent = 4) \n",
> +    "with open(\"sample.json\", \"w\") as f: \n",
> +    "    f.write(json_object) \n",
> +    "with open('sample.json', 'r') as f:\n",
> +    "    output = json.load(f)\n",
> +    "print(output)"
> The assignment was "Then let the user enter new entries (consisting of at least a title, an author, and an item type)", but other than that good.
> 
> In README.md <https://github.com/FrankieChik/python-for-the-humanities-homework/pull/13#discussion_r387731139>:
> 
> > +## The Idea of Abdication in the Bamboo-Slip Manuscripts and its Theoretical Deficiencies
> +
> +1. Introduction  
> +Why inheritance was the main method of achieving political succession in ancient China?  
> +Three Chu-script bamboo-slip manuscripts from the middle period of the Warring States: *Tang Yu zhi dao*, *Zigao*, and *Rongchengshi* unhesitatingly support the notion of abdication.  
> +
> +2. Definition of **Adication** in unearthed manuscripts. 
> +* *Tang Yu zhi dao*
> +  * *Tang Yu zhi dao*1
> +  * *Tang Yu zhi dao*2
> +* *Zigao*
> +  * *Zigao*1
> +  * *Zigao*2
> +
> +3. Conclusion  
> +<https://www.history.com/this-day-in-history/last-emperor-of-china-abdicates>
> usually, you use [link text](link) to create links.
> 
> In README.md <https://github.com/FrankieChik/python-for-the-humanities-homework/pull/13#discussion_r387732295>:
> 
> > +## The Idea of Abdication in the Bamboo-Slip Manuscripts and its Theoretical Deficiencies
> +
> +1. Introduction  
> +Why inheritance was the main method of achieving political succession in ancient China?  
> +Three Chu-script bamboo-slip manuscripts from the middle period of the Warring States: *Tang Yu zhi dao*, *Zigao*, and *Rongchengshi* unhesitatingly support the notion of abdication.  
> +
> +2. Definition of **Adication** in unearthed manuscripts. 
> +* *Tang Yu zhi dao*
> +  * *Tang Yu zhi dao*1
> +  * *Tang Yu zhi dao*2
> +* *Zigao*
> +  * *Zigao*1
> +  * *Zigao*2
> +
> +3. Conclusion  
> +<https://www.history.com/this-day-in-history/last-emperor-of-china-abdicates>
> and the assignment asked for multiple paragraphs of text. other than that good.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/FrankieChik/python-for-the-humanities-homework/pull/13?email_source=notifications&email_token=AOJI6IWJL74HDLMNRHY6CRLRFZWCTA5CNFSM4K35RXXKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCX6FSRQ#pullrequestreview-368859462>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AOJI6IUVIUNXKNDOQ5QPT7LRFZWCTANCNFSM4K35RXXA>.
>

jdamerow commented:
It doesn't seem to be in this pull request. Where did you commit them?

Commit:
03Homework-answers.ipynb

Commit:
Homework2(This_is_my_answers).ipynb

Commit:
Homework-06-documentation-testsanswers.ipynb

FrankieChik committed:
Create project.py

FrankieChik committed:
Update project.py

FrankieChik committed:
Create project_code_04082020.py

Hi Julia and Ken,
I went over the program historians and other online topic modeling resources these two days. I am finally able to tokenize and clean the texts I am now working on. I ran the code once on Pycharm but only it came out with some numbers like [(0, 24), (1, 21), (2, 6), (3, 3), (4, 3), (5, 3), (6, 3), (7, 3), (8, 3), (9, 3), (10, 3), (11, 3), (12, 3).......
I think I may need to process topic modeling this week but may I know if I am on the right track? Thanks.
